src/pool/srv_pool.c

```diff
@@ -2234,6 +2235,112 @@ transfer_cont_buf(struct daos_pool_cont_info *cont_buf, size_t ncont,
 	return rc;
 }
 
+/**
+ * Send CaRT RPC from mgmt svc (drpc) to pool svc to get container list.
+ *
+ * \param[in]	uuid		UUID of the pool
+ * \param[in]	ranks		Pool service replicas
+ * \param[out]	containers	Array of container information (allocated)
+ * \param[out]	ncontainers	Number of items in containers
+ *
+ * return	0		Success
+ *
+ */
+int
+ds_pool_svc_list_cont(uuid_t uuid, d_rank_list_t *ranks,
+		      struct daos_pool_cont_info **containers,
+		      uint64_t *ncontainers)
+{
+	int				rc;
+	struct rsvc_client		client;
+	crt_endpoint_t			ep;
+	struct dss_module_info		*info = dss_get_module_info();
+	crt_rpc_t			*rpc;
+	struct pool_list_cont_in	*in;
+	struct pool_list_cont_out	*out;
+	uint64_t			resp_ncont = 65536;
```

Comment:
Magic number?

Reply:
changed this to a smaller starting allocation length in latest patch